src/Compatibility.php

```diff
@@ -123,15 +124,15 @@ public static function addHiddenContext(string $key, mixed $value): void
      * @see https://github.com/laravel/framework/pull/49730
      * @see https://github.com/laravel/framework/releases/tag/v11.0.0
      */
-    public static function getHiddenContext(string $key): mixed
+    public static function getHiddenContext(string $key, mixed $default = null): mixed
```

Comment:
This API now matches Context's API where the second parameter can be a fallback value.